types/node/index.d.ts

```diff
@@ -497,9 +497,17 @@ declare namespace NodeJS {
         sorted?: boolean | ((a: string, b: string) => number);
     }
 
+    interface ConsoleConstructorOptions {
+        stdout: WritableStream,
```

Comment:
There are lint errors here -- should use `;` not `,`

Reply:
ups, seems I'm getting too fast with pointing to unrelated fails. should be fixed.